web/src/api/hfiCalculatorAPI.ts

```diff
@@ -71,3 +77,43 @@ export async function getDailies(
     last_updated: DateTime.fromISO(daily.last_updated)
   }))
 }
+
+export async function getHFIResult(
+  request: HFIResultRequest
+): Promise<HFIResultResponse> {
+  const { data } = await axios.post<RawHFIResultResponse>(baseUrl, {
+    ...request,
+    selected_prep_date: request.selected_prep_date?.toISOString().split('T')[0] // Infinite sadness
```

Comment:
Only used once, by maybe wrapping in function and explaining in detail what's going on here? We have so many time-zone conversations, I can only assume that whomever has to maintain this code will be lost without context.